ONSdigital/dp-observation-importer
# Release 1.4.0 - Add new health check

### What

Implement new health check endpoint, include checkers for kafka consumers and producers, graph database and dataset API.

Also includes:

* Update service dependency manager from go vendor to go modules
* Update dependencies to latest versions
* Use new log library, log.go instead of go-ns
* Use new dp-kafka library instead of go-ns
* Prevent application prematurely from shutting down on start up, and make sure service can reconnect to kafka if connection is lost

### How to review

* Check tests pass
* Check develop environment is working accordingly and consul outputs data from new health endpoint

### Who can review

Anyone
